uber/models.py

```diff
@@ -771,7 +771,6 @@ class Attendee(MagModel, TakesPaymentMixin):
     region        = Column(UnicodeText)
     country       = Column(UnicodeText)
     no_cellphone  = Column(Boolean, default=False)
```

Comment:
This wasn't actually being used, so I ripped it out.  I'll need to ask MAGStock if they want this, since I saw a reference to "Emergency Contact" and it wasn't clear if they just meant the number or if they also wanted the name.